packages/details/theme/lumo/vaadin-details-summary-styles.js

```diff
@@ -25,11 +25,12 @@ const detailsSummary = css`
     -webkit-tap-highlight-color: transparent;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
+    --_disabled-value-color: var(--vaadin-input-field-disabled-value-color, var(--lumo-disabled-text-color));
   }
 
   :host([disabled]),
   :host([disabled]) [part='toggle'] {
-    color: var(--lumo-disabled-text-color);
+    color: var(--_disabled-value-color);
     cursor: default;
```

Comment:
Details summary is not really a value, so I don't think the disabled value color property makes sense here.